From 5114212c64f4d761871e23b14432b3ee7d793418 Mon Sep 17 00:00:00 2001
From: Ming Chen <ming.a.chen@intel.com>
Date: Mon, 13 May 2019 21:58:58 +0800
Subject: [PATCH] Don't get mounted filesystems from /etc/fstab

/etc/fstab doesn't exist on Clear Linux by default. Use fstrim -av to trim all supported mounted filesystems rather than trimming filesystems from /etc/fstab.

Signed-off-by: Ming Chen <ming.a.chen@intel.com>
---
 sys-utils/fstrim.service.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sys-utils/fstrim.service.in b/sys-utils/fstrim.service.in
index 2d5daf9..d1c1f3f 100644
--- a/sys-utils/fstrim.service.in
+++ b/sys-utils/fstrim.service.in
@@ -1,7 +1,7 @@
 [Unit]
-Description=Discard unused blocks on filesystems from /etc/fstab
+Description=Discard unused blocks on filesystems
 Documentation=man:fstrim(8)
 
 [Service]
 Type=oneshot
-ExecStart=@sbindir@/fstrim -Av
+ExecStart=@sbindir@/fstrim -av
-- 
2.21.0

